.appveyor.yml

```diff
@@ -22,6 +22,13 @@ test_script:
   - npm ci
   - cargo test
   - cargo build --release -p wasm-bindgen-cli
+  - where chromedriver
+  - set CHROMEDRIVER=C:\Tools\WebDriver\chromedriver.exe
+  - cargo test -p js-sys --target wasm32-unknown-unknown
+  - cargo test -p web-sys
+  - cargo test -p web-sys --target wasm32-unknown-unknown
+  - cargo test -p webidl-tests
```

Comment:
The web-sys and webidl-tests on the native platform are unnecessary, and just take extra time building. These crates only have tests for `wasm32-unknown-unknown`.

Reply:
Great.  I suspected that was the case, but thanks for confirming!